plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.scss

```diff
@@ -0,0 +1,111 @@
+@import '../../stylesheets/_variables.scss';
+
+.woocommerce-marketplace {
+	&__product-card {
+		padding: $large-gap;
+		border-radius: $grid-unit-05 !important;
+		margin-top: $large-gap;
+
+		&__content {
+			display: grid;
+			grid-template-columns: 1fr;
+			align-items: flex-start;
+			gap: $medium-gap;
+			justify-content: space-between;
+		}
+
+		&__header {
+			grid-row: auto;
+			grid-column: 1;
```

Comment:
Don't think these lines are doing anything (`grid-row: auto` is the default value, and `grid-column: 1` is implied by the fact that there's only one column).

```suggestion
```